app/utilities/schema.py

```diff
@@ -46,17 +47,13 @@ def load_schema_file(schema_file, language_code=None):
 
 
 def _get_schema_file_path(schema_file, language_code):
-    if language_code != 'en':
-        schema_dir = os.path.join(settings.EQ_SCHEMA_DIRECTORY, language_code)
-    else:
-        schema_dir = settings.EQ_SCHEMA_DIRECTORY
-
+    schema_dir = os.path.join(settings.EQ_SCHEMA_DIRECTORY, language_code)
     return os.path.join(schema_dir, schema_file)
 
 
 def available_json_schemas():
     files = []
-    for file in os.listdir(settings.EQ_SCHEMA_DIRECTORY):
-        if os.path.isfile(os.path.join(settings.EQ_SCHEMA_DIRECTORY, file)) and file.endswith(".json"):
+    for file in os.listdir(settings.EQ_SCHEMA_DIRECTORY_DEFAULT):
```

Comment:
Changed it so the function is there and the env var is gone, but I don't know whether having `language_code='en'` seems right?